explorer/websocket.go

```diff
@@ -126,12 +132,6 @@ func safeClose(cc hubSpoke) {
 func (wsh *WebsocketHub) Stop() {
 	// end the run() loop, allowing in progress operations to complete
 	wsh.quitWSHandler <- struct{}{}
-	// end the buffer interval send loop
-	wsh.bufferTickerChan <- tickerSigStop
-	// unregister all clients
-	for client := range wsh.clients {
```

Comment:
This ranges over `wsh.clients` , is only called from the main go routine and so causes a data race. Moved it to the main run loop.